include/circt/Dialect/FIRRTL/FIRRTLOpInterfaces.td

```diff
@@ -277,6 +277,8 @@ def FConnectLike : OpInterface<"FConnectLike"> {
   ];
 }
 
+def InnerRefNamespace : NativeOpTrait<"InnerRefNamespace">;
```

Comment:
Can this be implemented in tablegen instead of a `NativeOpTrait`?  I blindly assume most things should be implemented that way if possible.

Reply:
AFAICT traits can't be done that way, only Interfaces.  Interfaces do define Traits as well I think, but I don't this makes sense as an "Interface", it's a mechanism to mix-in the InnerRef verif bits.